benefits/static/css/styles.css

```diff
@@ -144,6 +144,45 @@ footer {
   height: 75px;
 }
 
+.radio-container {
+  display: block;
+  margin-left: 15%;
+}
+
+.radio-label {
```

Comment:
Both the radio label and the radio input itself should have `cursor: pointer`, like https://github.com/uswds/uswds/commit/73ec4d8059af7b8ee67130e07a6bf21cace1c77a, so that when a Desktop user hovers their mouse over the 🔘 radio input or the radio label (the text), the mouse cursor turns into a 👆